processor/src/chiplets/hasher/mod.rs

```diff
@@ -410,9 +387,7 @@ impl Hasher {
         chiplets_bus: &mut ChipletsBus,
     ) -> AuxTraceBuilder {
         // provide all lookups to the ChipletsBus.
-        for lookup in self.lookups {
-            chiplets_bus.provide_hasher_lookup(lookup, lookup.cycle() as u32);
-        }
+        chiplets_bus.provide_hasher_lookups();
```

Comment:
We actually want to remove this completely, so this method will no longer need the `chiplets_bus` to be passed. Instead, the responses will all be provided at the same time the they are computed